src/MainWindow.cpp

```diff
@@ -232,7 +233,9 @@ void MainWindow::connectActions()
     // "Treemap" menu
 
     connect( _ui->actionShowTreemap, SIGNAL( toggled( bool )   ),
-	     this,		     SLOT  ( showTreemapView() ) );
+         this,		     SLOT  ( showTreemapView() ) );
+    connect( _ui->actionTreemapAsSidePanel, SIGNAL( toggled( bool )   ),
```

Comment:
As with all those lengthy connect() calls, there should be an empty line above and below; that makes them a lot easier to read.